.azure/templates/download-artifacts.yml

```diff
@@ -11,19 +11,19 @@ steps:
   inputs:
     artifactName: drop
     itemPattern: drop/${{ parameters.platform }}/${{ parameters.arch }}_*_${{ parameters.tls }}/**
-    downloadPath: $(Build.TempDirectory)
+    downloadPath: $(Agent.TempDirectory)
 
 - task: DownloadBuildArtifacts@0
-  displayName: Download�CLOG Artifacts
+  displayName: Download�CLOG Artifacts
```

Comment:
??? Undo this ???

Reply:
The original commit must have used a UTF-16 marker there or something, which VS Code got confusing on when saving. Fixed.